sdk/src/cameras/itof-camera/camera_itof.cpp

```diff
@@ -807,6 +823,47 @@ aditof::Status CameraItof::requestFrame(aditof::Frame *frame) {
     memcpy(reinterpret_cast<uint8_t *>(metadataLocation),
            reinterpret_cast<uint8_t *>(&metadata), sizeof(metadata));
 
+#ifdef TIME_PROFILING
+    steady_clock::time_point rf_t1 = steady_clock::now();
+#endif
+
+#ifdef TIME_PROFILING
+    // Measure execution times, display statistics
+    auto request_frame_elapsed_time =
+        std::chrono::duration_cast<std::chrono::microseconds>(rf_t1 - rf_t0);
+    auto compute_xyz_elapsed_time =
+        std::chrono::duration_cast<std::chrono::microseconds>(cxyz_t1 -
+                                                              cxyz_t0);
+    auto sensor_get_frame_elapsed_time =
+        std::chrono::duration_cast<std::chrono::microseconds>(sgf_t1 - sgf_t0);
+
+    auto total_profiled_time =
+        compute_xyz_elapsed_time + sensor_get_frame_elapsed_time;
+
+    std::cout << "CameraItof::requestFrame() took: "
```

Comment:
Why do you use std cout and not LOG(INFO) from glog?

Reply:
The main reason was to easily align the lines being printed. For example:
```
CameraItof::requestFrame() took: 80011us
|--> DepthSensorInterface::getFrame() took: 49724us(62.1465%)
|--> Algorithms::computeXYZ() took: 29566us(36.9524%)
Total profiled time: 79290us(99.0989%)
```
Another reason is that this way it's easier/clearer to separate the application usual logging from the printing of the time profiling. When one activates time profiling the most important thing to spot are the time measurements.
Also GLOG prints this header at the beginning of each line which is much noise in this situation.